src/kw_ssh.sh

```diff
@@ -109,6 +122,78 @@ function run_command_in_the_remote()
   cmd_manager "$flag" "$ssh_compose"
 }
 
+function ssh_send_file_to_remote()
+{
+  local user
+  local port
+  local remote
+  local remote_file
+  local remote_file_host
+  local send_path
+  local to_path
+  local flag
+  local cmd
+  local send_option
+
+  user="${remote_parameters['REMOTE_USER']}"
+  port="${remote_parameters['REMOTE_PORT']}"
+  remote="${remote_parameters['REMOTE_IP']}"
+  remote_file="${remote_parameters['REMOTE_FILE']}"
+  remote_file_host="${remote_parameters['REMOTE_FILE_HOST']}"
+  send_path=${options_values['SEND']}
+  to_path=${options_values['TO']:-'.'}
+  flag=${options_values['VERBOSE']}
+  item_type=${item_type:-''}
+
+  if [[ ! -f "$send_path" ]]; then
+    item_type="-r"
+  fi
+
+  if [[ -f "$remote_file" ]]; then
+    cmd="scp -F ${remote_file} ${item_type} ${send_path} ${remote_file_host}:${to_path}"
+  else
+    cmd="scp -P ${port} ${item_type} ${send_path} ${user}@${remote}:${to_path}"
+  fi
+
+  cmd_manager "$flag" "$cmd"
```

Comment:
This command can fail, and you need to handle that by providing a meaningful message in case of failure.